ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -451,7 +451,9 @@ public struct Avatar: View {
 }
 
 /// Properties available to customize the state of the avatar
-class MSFAvatarStateImpl: NSObject, ObservableObject, MSFAvatarState {
+class MSFAvatarStateImpl: NSObject, ObservableObject, Identifiable, MSFAvatarState {
```

Comment:
Since this is already in main, the PR should target to merge to main instead of vnext-prototype.

Reply:
Also, main already has the new demo controller based on the UITableView after the merge of #674.
Can you also bring those changes to your PR branch and test in the new demo controller? (updating the description screenshots/videos)